cypress/support/slugs.js

```diff
@@ -0,0 +1,20 @@
+var news_slugs = [
```

Comment:
This is good example of what fixture should be. Move this test data into fixtures and update import function in the e2e test.

Reply:
Thank you, done! :)